Created Grantable Customer Story (#13495)
PostHog/posthog.com: contents/customers/grantable.md

<BorderWrapper>
<Quote
 imageSource="https://res.cloudinary.com/dmukukwp6/image/upload/evan_rallis_08cffd54f7.jpeg"
    size="md"
    name="Evan Rallis"
    title="Head of Product & Growth"
    quote={`“We’ve started adding PostHog events into every PR. All new features have a PostHog event requirement,” Evan says. “We track usage, pipe that into dashboards for activation and retention, and now we’re starting to act on it automatically with Workflows.”`}
/>
</BorderWrapper>

That move, from ‘see data’ to ‘do something with it right away,’ is exactly why Grantable is betting on Workflows.

## PostHog Workflows vs. other tools 
Grantable had already tried basically every workflow automation platform in the market – tools like Zapier, Make, and relay.app – and had gone all-in on them before: hiring teams to build complex automation, shipping real production flows on top of those systems, even auto-generating large sites with tens of thousands of pages powered by those workflows.

However, this didn’t work for two reasons.

### No shared source of truth

> Tools like Zapier or Make don’t sit on top of your product data. You have to keep feeding them user state, usage events, subscription info, ICP classification logic, etc. Every time your data model changes, you’re updating brittle queries and reconnecting fields by hand.

Reviewer: ```suggestion
Tools like Zapier or Make don’t sit on top of your product data. You must continually provide them with user states, usage events, subscription information, ICP classification logic. Every time your data model changes, you’re updating brittle queries and reconnecting fields by hand.
```
